src/Tasks/Common/src/ConflictResolution/FrameworkListReader.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.NET.Build.Tasks;
+using Microsoft.Build.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Xml.Linq;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    static class FrameworkListReader
+    {
+        public static IEnumerable<ConflictItem> LoadConflictItems(string frameworkListPath, ILog log)
+        {
+            if (frameworkListPath == null)
+            {
+                throw new ArgumentNullException(nameof(frameworkListPath));
+            }
+
+            if (!File.Exists(frameworkListPath))
+            {
+                //  This is not an error, as we get both the root target framework directory as well as the Facades folder passed in as TargetFrameworkDirectories.
+                //  Only the root will have a RedistList\FrameworkList.xml in it
+                yield break;
```

Comment:
Consider not using a yield enumerator if this is a hot path where allocations matter.